fast-list.js

```diff
@@ -112,7 +117,7 @@ FastList.prototype = {
     var itemPerScreen = geo.viewportHeight / geo.itemHeight;
     // Taking into account the will-change budget multiplier from
     // layout/base/nsDisplayList.cpp#1193
-    geo.maxItemCount = Math.floor(itemPerScreen * 2.8);
+    geo.maxItemCount = Math.floor(itemPerScreen * 3);
```

Comment:
sneacked this one in.
since the will-change warning is now fixed I was able to see that it's not causing any trouble and it should help a tiny bit with the checkerboarding.